src/main/resources/static/js/alert.js

```diff
@@ -0,0 +1,7 @@
+$(function() {
```

Comment:
These various js files came along with the theme, right? Can you put them in a sub-directory to emphasize that they are not typically for us to maintain? Ideally these would have been pulled via a webjar and we wouldn't need to source control them in the first place. Any possibility of that? 

With that said, these also don't look very "angular". It's an artificial line to draw, but I'd really like to stick to Angular-centric widgets/modules, such as those available at

http://ngmodules.org/